### PR TITLE
feat: update SDK based on new verification model

### PR DESCRIPTION
**Pull request by xxuejie**
Please hold on merging this for now till the PR in CKB repo is merged, tho we can get review process started early.

### CODE REVIEW

**Comment by classicalliu** [comment]
CI failed

**Comment by ashchan** [comment]
Is this going to `v0.14.0`?

**Comment by xxuejie** [comment]
> Is this going to `v0.14.0`?

That's the plan.

**Comment by ashchan** [comment]
BTW which is the upstream CKB pr on this change?

**Comment by xxuejie** [comment]
> BTW which is the upstream CKB pr on this change?

Still working on it, ETA is this morning

**Comment by ashchan** [comment]
* https://github.com/nervosnetwork/ckb/pull/913

File: lib/ckb/api.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def initialize(host: CKB::RPC::DEFAULT_URL, mode: MODE::TESTNET)
       @rpc = CKB::RPC.new(host: host)
       if mode == MODE::TESTNET
         # Testnet system script code_hash
-        expected_code_hash = "0x9e3b3557f11b2b3532ce352bfe8017e9fd11d154c4c7f9b7aaaa1e621b539a08"
+        expected_code_hash = "0xa4a3d87cd69733562865ddfaf12dd44cf4cd396c4128ff1a5839c2fe48b2a1a1"
```
Comment (ashchan) [inline lib/ckb/api.rb]: This seems to be changing again on CKB `rc/v0.14.0`. Confirming that now.

https://github.com/nervosnetwork/ckb/pull/941/files#diff-346487b1ea6102562958846069e090deR9

**Pull request merged**